html/semantics/interactive-elements/the-dialog-element/non-modal-dialog-layout.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<link rel=author href="mailto:jarhar@chromium.org">
```

Comment:
nit: Here and other tests, would  be nice if double quotes were used around the attr value.

Reply:
I've been doing `rel=author` instead of `rel="author"` for years, is there a style guide or anything? I kind of like not using quotes with simple string values.

Reply:
not sure if there's a style guide (I wish there was), but attribute values most commonly have double quotes around them, not sure why smaller values should be exempt.

Reply:
ok, ill add quotes to all the dialog tests as a followup

Reply:
I created a followup to do this: https://github.com/web-platform-tests/wpt/pull/32043